Update/toki_003.txt

```diff
@@ -2770,13 +2770,14 @@ void main()
 //rそして、大石は翌日、父のもと上司であるこの山沖を紹介してくれた。それだけに彼女は彼らに対して、全幅の信頼を置いていたのだ。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "そして、大石は翌日、父のもと上司であるこの山沖を紹介してくれた。それだけに彼女は彼らに対して、全幅の信頼を置いていたのだ。",
-			NULL, "And the next day, Ooishi introduced her to Yamaoki, who was her father's boss. From that point on, she completely placed her trust in them.", Line_Normal);
+			NULL, "And the next day, Ooishi introduced her to Yamaoki, who was her father's boss. That just went to show how they had full confidence in her.", Line_Normal);
 	ClearMessage();
 
 //r警察官への道を薦めてくれたのも、他ならぬ山沖だ。誰かをあてにするのではなく、自ら運命を切り開くのも手ではないか、…と。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "警察官への道を薦めてくれたのも、他ならぬ山沖だ。誰かをあてにするのではなく、自ら運命を切り開くのも手ではないか、…と。",
-			NULL, "And the one who encouraged her to become a police officer was none other than Yamaoki. Instead of relying on others, she wanted carve out her fate with her own hands...", Line_Normal);
+			NULL, "And the one who encouraged her to become a police officer was none other than Yamaoki. He asked if instead of relying on others, she wanted to
```

Comment:
```suggestion
			NULL, "And the one who encouraged her to become a police officer was none other than Yamaoki. He asked if instead of relying on others, she wanted to carve out her fate with her own hands...", Line_Normal);
```